src/packages/admin-ui-components/src/detail-panel/component.tsx

```diff
@@ -71,9 +71,11 @@ const getField = ({ field }: { field: EntityField }) => {
 	}
 
 	const { enumByName } = useSchema();
-	const enumField = enumByName(field.type);
+	const enumField = enumByName(field.type.replace('[]', ''));
 	if (enumField) {
-		return <EnumField name={field.name} typeEnum={enumField} />;
+		return (
+			<EnumField name={field.name} typeEnum={enumField} multiple={field.type.endsWith('[]')} />
```

Comment:
Depending on whether we add the `isArray` flag here this might be `field.isArray`

Reply:
Fixed. Added new property called `isArray`.